### PR TITLE
Add Bhavya Wahie to the list

### PR DESCRIPTION
**Pull request by Bhavyawahie**
### Public/Data updated

A new JSON with the name `bhavyawahie.json` added to the `public/data` directory to incorporate for a new user linkFree profile.
The above chaged can be experienced at http://linkfree.eddiehub.org ✌

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/333"></a>

### CODE REVIEW

File: public/data/bhavyawahie.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Bhavya Wahie",
+    "bio": "Founder of EddieHub",
+    "avatar": "https://github.com/bhavyawahie.png",
+    "links": [
+    {
+        "name": "Follow here to see me talk in memes😎!",
```
Comment (CodingSpecies) [inline public/data/bhavyawahie.json:7]: @Bhavyawahie Please add a space between the emoji and the word `memes`. Thank you

File: public/data/bhavyawahie.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://twitter.com/bhavyawahie",
+        "icon": "twitter"
+    }, 
+    {
+        "name": "Follow and See my recent side projects on GitHub👩‍💻! ",
```
Comment (CodingSpecies) [inline public/data/bhavyawahie.json:12]: @Bhavyawahie Please add a space between the emoji and the word `GitHub`. Thank you

File: public/data/bhavyawahie.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://github.com/bhavyawahie",
+        "icon": "github"
+    },
+    {
+        "name": "Stalk me on instagram👀",
```
Comment (CodingSpecies) [inline public/data/bhavyawahie.json:17]: @Bhavyawahie Please add a space between the emoji and the word `Instagram`, also capitalise Instagram please. Thank you

Comment (CodingSpecies) [inline public/data/bhavyawahie.json:17]: Also, would it be possible is you put `follow me/ join me on instagram`? Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/bhavyawahie.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/bhavyawahie.json:7]:
```suggestion
        "name": "Follow here to see me talk in memes 😎",
```

File: public/data/bhavyawahie.json (hunk 2 above)
Comment (Panquesito7) [inline public/data/bhavyawahie.json:12]:
```suggestion
        "name": "Follow and See my recent side projects on GitHub 👩‍💻 ",
```

File: public/data/bhavyawahie.json (hunk 3 above)
Comment (Panquesito7) [inline public/data/bhavyawahie.json:17]:
```suggestion
        "name": "Follow me on Instagram 👀",
```

File: public/data/bhavyawahie.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://instagram.com/bhavyawahie",
+        "icon": "instagram"
+    }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/bhavyawahie.json:22]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/bhavyawahie.json (hunk 1 above)
Comment (Bhavyawahie) [reply public/data/bhavyawahie.json:7]: Sure, I'll make the required changes 😊

File: public/data/bhavyawahie.json (hunk 2 above)
Comment (Bhavyawahie) [reply public/data/bhavyawahie.json:12]: Sure, I'll make the required changes 😊

File: public/data/bhavyawahie.json (hunk 3 above)
Comment (Bhavyawahie) [reply public/data/bhavyawahie.json:17]: Sure, I'll make the required changes 😊

**Review by Bhavyawahie — COMMENTED** [review]
Made the suggested changes!

File: public/data/bhavyawahie.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Bhavya Wahie",
+    "bio": "Full stack Developer with an eye for design.",
```
Comment (Panquesito7) [inline public/data/bhavyawahie.json:3]:
```suggestion
    "bio": "Full-stack Developer with an eye for design",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**